Jenkinsfile

```diff
@@ -263,6 +263,9 @@ try {
         }
       } else {
         githubNotify context: 'CI Pipeline', status: 'FAILURE'
+        if (env.BRANCH_NAME == 'master' || full_ci) {
+          githubNotify context: 'Full CI', status: 'FAILURE'
+        }
```

Comment:
This has nothing directly to do with the jemalloc change I assume, it's just added in here for convenience?

Reply:
Not for convenience. Previously, the "Full CI passed" status was never reset. Not a problem, because the other one failed, but still confusing.